README.md

```diff
@@ -1,3 +1,11 @@
 # BSP layer for updating Raspberrypi with OSTree
 
 OSTree bootloader integration for Raspberry Pi. Add it to BBLAYERS when using [meta-updater](https://github.com/advancedtelematic/meta-updater) with [Raspberry Pi](https://github.com/agherzan/meta-raspberrypi).
+Wi-Fi enablement for Raspberry Pi
```

Comment:
I think you need another linebreak before this line. When github renders it, it gets mashed into the preceding line, which seems weird.

Reply:
Yes, maybe,  bulleted lists would be even better.